[2023-10-20 23:28:20,290] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-09-15T00:00:00+00:00 [queued]>
[2023-10-20 23:28:20,311] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-09-15T00:00:00+00:00 [queued]>
[2023-10-20 23:28:20,312] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:20,314] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:28:20,314] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:20,352] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2023-09-15 00:00:00+00:00
[2023-10-20 23:28:20,360] {standard_task_runner.py:52} INFO - Started process 485 to run task
[2023-10-20 23:28:20,366] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2023-09-15T00:00:00+00:00', '--job-id', '1247', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmprgtikcaj', '--error-file', '/tmp/tmpt8aiobjw']
[2023-10-20 23:28:20,369] {standard_task_runner.py:80} INFO - Job 1247: Subtask tercera_tarea
[2023-10-20 23:28:20,525] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-09-15T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:28:20,652] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-09-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-15T00:00:00+00:00
[2023-10-20 23:28:20,654] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:28:20,656] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-10-20 23:28:20,679] {subprocess.py:85} INFO - Output:
[2023-10-20 23:28:20,682] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-10-20 23:28:20,686] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:28:20,753] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20230915T000000, start_date=20231020T232820, end_date=20231020T232820
[2023-10-20 23:28:20,822] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:28:20,970] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
